src/build_and_deploy.sh

```diff
@@ -0,0 +1,34 @@
+#
+# The `build_and_deploy.sh` file centralizes functions related to kw bd.
+#
+
+include "${KW_LIB_DIR}/deploy.sh"
+include "${KW_LIB_DIR}/build.sh"
+
+# kw bd is a shortcut to build and deploy a new kernel.
+#
+# @raw_options String with all deploy options
+#
+function build_and_deploy_main()
+{
+
+  local raw_options="$*"
+
+  if [[ "$1" =~ -h|--help ]]; then
+    build_and_deploy_help "$1"
+    exit 0
+  fi
+
+  build_kernel_main && deploy_main 1 "$raw_options"
```

Comment:
It is better to call each function, check the return type, and then execute the next function.